spec/fixtures/files/bulk_update/trainee_uploads/failed_with_failed.csv

```diff
@@ -6,10 +6,10 @@ Non-UK degree type must be entered if specifying a previous non-UK degree;
 Training Route has invalid reference data value of '13'. Valid values are '02', '03', '09', '10', '11', '12', '14'.;
 Trainee disabilities attributes contain duplicate values"
 24/25-82,,'5205630693043,Abel,Hills,Halvorson,'2002-01-07,,'10,Abel.Hills@example.com,GB,,,,,,,,,,,'202,'11,'032,'100358,,,'01,'2024-10-01,'2025-07-14,'13911,2024,,,'2024-10-01,'2024-08-13,'2,'6,,,,,,,,,,,,,Validation passed,
-24/25-85,,'7982579851416,Richie,Welch,Christiansen,'1993-04-19,,'99,Richie.Welch@example.com,GB,,,,,,,,,,,'202,'13,,'100302,,,'01,'2024-10-01,'2023-07-03,'13915,2024,,,'2024-10-01,'2024-08-16,'2,'6,,,,,,,,'100184,,,,,6 errors found,"Degree graduation year must be entered if specifying a previous UK degree or non-UK degree;
+24/25-85,,'7982579851416,Richie,Welch,Christiansen,'1993-04-19,,'99,Richie.Welch@example.com,GB,,,,,,,,,,,'202,'13,,'100302,,,'01,'2024-10-01,'2023-07-03,'13915,2024,,,'2024-10-01,'2024-08-16,'2,'6,,,,,,,,'100184,,,,,6 errors found,"ITT End Date must be after ITT start date;
+Degree graduation year must be entered if specifying a previous UK degree or non-UK degree;
 UK degree type must be entered if specifying a previous UK degree;
 Degree grade must be entered if specifying a previous UK degree or non-UK degree;
 Qualification Aim must be entered if 202 selected for ITT Aim;
-Training Route has invalid reference data value of '13'. Valid values are '02', '03', '09', '10', '11', '12', '14'.;
-ITT End Date must be after ITT start date"
+Training Route has invalid reference data value of '13'. Valid values are '02', '03', '09', '10', '11', '12', '14'."
 24/25-88,,'1423427395203,Scott,Hermiston,Beier,'1999-03-10,,'10,Scott.Hermiston@example.com,GB,,,,,,,,,,,'202,'12,'021,'100891,,,'64,'2024-10-01,'2025-07-15,'13911,2024,,,'2024-10-01,'2024-08-28,'2,'6,,,,,,,,,,,,,1 error found,UK degree type or Non-UK degree type must be entered if specifying a postgraduate Training Route
```

Comment:
the validation ordering changed slightly (no changes to the actual wording of them)